AuroraStudiosDigipen/Borealis
# Project management

Created a Project interface in Editor Project:
Important Functions:
LoadProject() -> Will open up a windows dialog box to look for a ".brproj" file which contains important information about the project
NewProject() -> Will open up a windows dialog box to specify a project name, will create a project folder with all necessary directories at the location of the directory in the windows dialog box

To test:
- In level editor, click "File" -> "New Project" and create a new Project folder: Content Browser Panel should be emptied(because there are no assets). Create a random file into the assets folder of the new project folder and it should show the file
- Reopen the level editor, click "File" -> "Open Project" and select the previously created Project Folder's .brproj file. It should load the project. Test Content Browser Panel as before.

Possible improvements:
Set default folder when opening up any windows dialog box based on directories(similar to Maya)
